benefits/core/templates/core/includes/form.html

```diff
@@ -3,14 +3,10 @@
 
   {% url form.action_url as form_action %}
 
-  <form id="{{ form.id }}"
-        class="{{ form.classes }}"
-        action="{{ form_action }}"
-        method="{{ form.method | default:"post" | upper }}"
-        role="form">
+  <form id="{{ form.id }}" action="{{ form_action }}" method="{{ form.method | default:"post" | upper }}">
     {% csrf_token %}
 
-    <div class="form-field-container d-flex flex-column gap-4">
```

Comment:
Decided to remove `d-flex flex-column gap-4` from the Form template. The Form template should be as generic as possible and not assume a design.